python/BBScraping.py

```diff
@@ -6,38 +9,54 @@
 import sys
 import os
 import time
+"""
+    This file is for blackboard scraping using python 
+"""
 
-# Username & Password
+""" 
+    setup username & password
+"""
 USER = sys.argv[1]
 USERNAME = sys.argv[2]
 PASSWORD = sys.argv[3]
 
-# connection to MongoDB
+"""
+    connection to MongoDB
+"""
 client = MongoClient('localhost', 27017)
```

Comment:
@alicechen1998  you need to delete line 26 and 27 since its already in line29 and 30

Reply:
sloved

Reply:
hey @alicechen1998 ran some tests and seems like there are some issues with running line 11?
![image](https://user-images.githubusercontent.com/42022911/143785458-c16f21f5-e2aa-48ab-a296-673f86a31aab.png)
if the list should be getting and data before running this python file, it might be good idea to include comment about how to run this for the rest of the team.

thx!

Reply:
I think @bhusalanil121 wrote this file. You can ask him about this. I just did some function header for this file